src/base/preferences.cpp

```diff
@@ -1410,64 +1410,14 @@ void Preferences::setTransHeaderState(const QByteArray &state)
 }
 
 //From old RssSettings class
-bool Preferences::isRSSEnabled() const
+bool Preferences::isRSSWidgetVisible() const
 {
-    return value("Preferences/RSS/RSSEnabled", false).toBool();
+    return value("GUI/RSSWidget/Visible", false).toBool();
```

Comment:
Comment about consistency: IIRC the rest of the settings(in settingstorage too) use "Enabled" to denote the visibility of gui element. I am just mentioning it. Do as you please here. It isn't a hard requirement for me.